book/controller.rst

```diff
@@ -438,7 +438,7 @@ If you want to redirect the user to another page, use the ``redirectToRoute()``
         return $this->redirectToRoute('homepage');
 
         // redirectToRoute is equivalent to using redirect() and generateUrl() together:
-        // return $this->redirect($this->generateUrl('homepage'), 301);
```

Comment:
I'm not sure this is a typo. The default value of the second argument of `redirect` is `302`:

```php
    protected function redirect($url, $status = 302)
    {
        return new RedirectResponse($url, $status);
    }
```